conbench/entities/run.py

```diff
@@ -12,6 +12,7 @@ class Run(Base, EntityMixin):
     __tablename__ = "run"
     id = NotNull(s.String(50), primary_key=True)
     name = Nullable(s.String(250))
+    reason = Nullable(s.String(250))
```

Comment:
I don't think there is, but is there value in validating the reasons? Ideally there would be a relatively small number of reasons and it isn't a free-form field (that's more or less what name is for...).

Maybe describing that in the docs is good enough?

Reply:
good point; I'll flesh out the docs a little more. I don't want to be prescriptive about what possible reasons could be, but I'll include a note like "runs with the same reason will probably be compared with each other so try to keep the cardinality down"